src/main/java/com/minecolonies/apiimp/initializer/ModBuildingsInitializer.java

```diff
@@ -123,6 +128,7 @@ public static void init(final RegistryEvent.Register<BuildingEntry> event)
                                    .setRegistryName(new ResourceLocation(Constants.MOD_ID, ModBuildings.COMPOSTER_ID))
                                    .addBuildingModuleProducer(() -> new ItemListModule(COMPOSTABLE_LIST), () -> new ItemListModuleView(COMPOSTABLE_LIST, COM_MINECOLONIES_REQUESTS_COMPOSTABLE_UI, false,
                                      (buildingView) -> IColonyManager.getInstance().getCompatibilityManager().getCompostInputs()))
+                                   .addBuildingModuleProducer(MinimumStockModule::new, MinimumStockModuleView::new)
```

Comment:
this one also interferes with regular working.

Reply:
Removed